src/ts/lib/dummy-instantiation.ts

```diff
@@ -1,26 +1,26 @@
 import { constants } from "ethers";
 import {
-  SafeCreationSettings,
-  VirtualTokenCreationSettings,
+    SafeCreationSettings,
+    VirtualTokenCreationSettings,
 } from "../proposal";
 import { BridgeParameter } from "./common-interfaces";
 
 export const dummyVirtualTokenCreationSettings: VirtualTokenCreationSettings = {
-  merkleRoot: constants.HashZero,
-  usdcToken: constants.AddressZero,
-  gnoToken: constants.AddressZero,
-  gnoPrice: "0",
-  wrappedNativeToken: constants.AddressZero,
-  nativeTokenPrice: "0",
+    merkleRoot: constants.HashZero,
+    usdcToken: constants.AddressZero,
+    gnoToken: constants.AddressZero,
+    gnoPrice: "0",
+    wrappedNativeToken: constants.AddressZero,
+    nativeTokenPrice: "0",
 };
 
 export const dummyteamConrollerSettings: SafeCreationSettings = {
-  owners: [3].map((i) => "0x".padEnd(42, i.toString())),
-  threshold: 1,
+    owners: [3].map((i) => "0x".padEnd(42, i.toString())),
+    threshold: 1,
 };
 
 export const dummyBridgeParameters: BridgeParameter = {
-  multiTokenMediatorGnosisChain: "0x" + "01".repeat(20),
-  multiTokenMediatorETH: "0x" + "02".repeat(20),
-  arbitraryMessageBridgeETH: "0x" + "03".repeat(20),
+    multiTokenMediatorGnosisChain: "0x" + "01".repeat(20),
+    multiTokenMediatorETH: "0x" + "02".repeat(20),
+    arbitraryMessageBridgeETH: "0x" + "03".repeat(20),
 };
```

Comment:
I'm quite surprised this change is ok for linting.
Indeed, that's because we have a bad ignore pattern for `"lib/"` in `.eslintrc.js`, if we change it to `/lib/` it works as expected and gives a few warnings, none important.
Not saying you should change it now, I'll open another PR. However you might want to checkout back this file to what it was before to avoid conflicts.